data/174/607/208/9/1746072089.geojson

```diff
@@ -0,0 +1,59 @@
+{
+  "id": 1746072089,
+  "type": "Feature",
+  "properties": {
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"uuuu",
+    "geom:area":0.000485,
+    "geom:bbox":"46.605075,24.560144,46.64456,24.58852",
+    "geom:latitude":24.573682,
+    "geom:longitude":46.62495,
+    "iso:country":"SA",
+    "lbl:latitude":24.575459,
+    "lbl:longitude":46.623604,
+    "mps:latitude":24.575459,
+    "mps:longitude":46.623604,
+    "mz:hierarchy_label":1,
+    "mz:is_current":1,
+    "mz:is_funky":0,
+    "name:ara_x_preferred":[
+        "\u0627\u0644\u0633\u0648\u064a\u062f\u064a \u0627\u0644\u063a\u0631\u0628\u064a"
+    ],
+    "name:eng_x_preferred":[
+        "Suwaidi Al Gharbi"
+    ],
+    "name:eng_x_variant":[
```

Comment:
3rd example of incorrect Arabic in English alt name slot.

Reply:
fixed in 27ea53d